packages/charting/src/components/BenchmarkBarChart/BenchmarkBarChart.style.ts

```diff
@@ -0,0 +1,37 @@
+import { IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle } from './BenchmarkBarChart.types';
+
+export const getBenchmarkBarChartStyles = (props: IBenchmarkBarChartStyleProps): IBenchmarkBarChartStyle => {
```

Comment:
Pass in theme. This will allow you to get the fonts more easily that Raghu mentioned below. Theme is piped in by the styled() HoC already so all you need to do is add a prop on IBenchmarkBarChartStyleProps and IBenchmarkBarChartProps of type ITheme and ensure it's provided to this function by getClassNames

Reply:
Thanks for your suggestion. Add a theme prop on IBenchmarkBarChartStyleProps and IBenchmarkBarChartProps.